src/config/mongoose_config_parser_toml.erl

```diff
@@ -130,7 +130,7 @@ auth_option_placement(<<"sasl_mechanisms">>) -> outer;
 auth_option_placement(_) -> inner.
 
 process_general(<<"loglevel">>, V) ->
-    #local_config{key = loglevel, value = V};
+    #local_config{key = loglevel, value = b2a(V)};
```

Comment:
I understand here that we're accepting only the binary then, right? While I totally agree with that and prefer the keywords rather than their number counterparts, I think this will need to be explicitly stated later on for the release. This is just to raise awareness and make some final decision later and document it properly, you don't need to take any action here.

Reply:
Yes.

Reply:
The `4 = error` trap caught me off guard and I am sure it would catch others as well. So we definitely need to make everything clear in the docs.